sql/information_schema/information_schema.go

```diff
@@ -2027,6 +1932,27 @@ func partitionKey(tableName string) []byte {
 	return []byte(InformationSchemaDatabaseName + "." + tableName)
 }
 
+// getColumnDefaultValue returns value for column default value in string format or nil for 'NULL' default value.
+func getColumnDefaultValue(cd *ColumnDefaultValue) interface{} {
```

Comment:
Don't do this, use expression.Wrapper to encode a null column default. Check out how it's used in the resolve_defaults analyzer rule.